### PR TITLE
Add serviceName question for user research studios

### PR DESCRIPTION
**Pull request by allait**
Sets serviceName to be the first question so that every created
draft has a name to display in the service list, similar to G7.

Question is copied from the current prototype.

### CODE REVIEW

File: frameworks/digital-outcomes-and-specialists/questions/services/serviceName.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+question: What is the name of studio?
```
Comment (TheDoubleK) [inline frameworks/digital-outcomes-and-specialists/questions/services/serviceName.yml]: Should it be "What is the name of **the** studio"?

File: frameworks/digital-outcomes-and-specialists/questions/services/serviceName.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+type: text
+hint: A studio is the room in which the participant(s) and moderator(s) carry out testing. The name will help people buyers refer to the right studio when communicating with you.
```
Comment (TheDoubleK) [inline frameworks/digital-outcomes-and-specialists/questions/services/serviceName.yml]: This should be "people" or "buyers", but not "people buyers".

Unless we're doing a framework for human trafficking? :trollface:

**Comment by allait** [comment]
@TheDoubleK good call, I've copied the text from the prototype and haven't checked it. Updated.

**Comment by quis** [comment]
:+1:

**Pull request merged**